src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Nav } from "react-bootstrap";
+import classNames from "classnames";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useUser } from "hooks/useUser";
+import { useAdminVenues } from "hooks/useAdminVenues";
+
+import { Venue_v2 } from "types/venues";
+
+import AdvancedSettings from "pages/Admin/AdvancedSettings";
+import EntranceExperience from "pages/Admin/EntranceExperience";
+import VenueWizard from "pages/Admin/Venue/VenueWizard/VenueWizard";
+import { LoadingPage } from "components/molecules/LoadingPage";
+
+import "./AdminAdvancedSettings.scss";
+
+export enum AdminAdvancedTab {
+  basicInfo = "basic_info",
+  entranceExperience = "entrance_experience",
+  advancedMapSettings = "advanced_map_settings",
+}
+
+const adminAdvancedTabs: Record<AdminAdvancedTab, String> = {
+  [AdminAdvancedTab.basicInfo]: "Start",
+  [AdminAdvancedTab.entranceExperience]: "Entrance",
+  [AdminAdvancedTab.advancedMapSettings]: "Advanced",
+};
```

Comment:
For a "real" constant object:
```suggestion
const adminAdvancedTabs: Record<AdminAdvancedTab, String> = Object.freeze({
  [AdminAdvancedTab.basicInfo]: "Start",
  [AdminAdvancedTab.entranceExperience]: "Entrance",
  [AdminAdvancedTab.advancedMapSettings]: "Advanced",
});
```

Reply:
The object is not being mutated or modified anywhere, I understand that this is a 'secure' way to ensure that this is not going to be ever changed, but maybe we should be using `readonly` instead of ES5 functions. We also use immutability whenever we can.

Reply:
`Record<AdminAdvancedTab, String>` can be then changed to `Readonly<Record<AdminAdvancedTab, String>>` as a signal for "immutability".

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/0d4d60cd9f66fb4682d6f3cbf71193467e5c5a52